src/keyValue.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, USC
```

Comment:
Generally avoid camel case in file names.

Reply:
fixing in next commit